### ISSUE TITLE
Need to add automatic quantity increaser/decreaser in cart section by using +/- 

### ISSUE THREAD

**Issue by singhrishabh93** [singhrishabh93/Hacktoberfest-2022#5]
Need to add automatic quantity increaser/decreaser in cart section by using +/- so that users can easily increase the quantity of the product which they want to purchase. 
There should be remove button which will be used to remove the product from the cart.

Remember create pull requests only after 1 October.

Happy Contribution 👍

**Comment by iAdityaNarayanT** [comment]
Hey there! I would like to work on this issue!

**Comment by singhrishabh93** [comment]
> Hey there! I would like to work on this issue! 

Okkk

**Comment by singhrishabh93** [comment]
@iAdityaNarayanT 
What's the progress?

**Comment by singhrishabh93** [comment]
@iAdityaNarayanT What's the progress?

**Comment by AbhishekTiwari23** [comment]
@singhrishabh93 I am interested. Please assign it to me; I will give my best.

**Comment by singhrishabh93** [comment]
> @singhrishabh93 I am interested. Please assign it to me; I will give my best.

Okay

**Comment by singhrishabh93** [comment]
> @singhrishabh93 I am interested. Please assign it to me; I will give my best.

Check I've assigned to you

**Comment by AbhishekTiwari23** [comment]
> > @singhrishabh93 I am interested. Please assign it to me; I will give my best.
> 
> Check I've assigned to you

Thanks, I will give my best. Is there any time restriction of 1 or 2 days?

**Comment by singhrishabh93** [comment]
> > > @singhrishabh93 I am interested. Please assign it to me; I will give my best.
> > 
> > Check I've assigned to you
> 
> Thanks, I will give my best. Is there any time restriction of 1 or 2 days?

@AbhishekTiwari23 No

**Comment by AbhishekTiwari23** [comment]
@singhrishabh93 I have made all the necessary changes, and soon I will make a pr

**Comment by singhrishabh93** [comment]
> @singhrishabh93 I have made all the necessary changes, and soon I will make a pr

@AbhishekTiwari23 Okay the make your PR

**Comment by AbhishekTiwari23** [comment]
Is this repo got excluded from hacktoberfest? Because I made a PR and it is showing excluded on hacktoberfest page

**Comment by singhrishabh93** [comment]
> Is this repo got excluded from hacktoberfest? Because I made a PR and it is showing excluded on hacktoberfest page

Can you figure it out for me why this project is excluded from hacktoberfest 2022

**Comment by AbhishekTiwari23** [comment]
<img width="1440" alt="Screenshot 2022-10-05 at 7 21 26 PM" src="https://user-images.githubusercontent.com/97469132/194077708-0f6e405a-6714-4a31-8d97-796e4a21e637.png">

Please check. There is one more repo. When I contacted his owner, he mentioned that he contacted hacktoberfest regarding that. Please make sure that this repo is a part of the fest

**Comment by singhrishabh93** [comment]
> <img width="1440" alt="Screenshot 2022-10-05 at 7 21 26 PM" src="https://user-images.githubusercontent.com/97469132/194077708-0f6e405a-6714-4a31-8d97-796e4a21e637.png">
> 
> Please check. There is one more repo. When I contacted his owner, he mentioned that he contacted hacktoberfest regarding that. Please make sure that this repo is a part of the fest

@AbhishekTiwari23 How to make this repo as a part of hacktoberfest 2023

**Comment by ItzLevvie** [comment]
There is a commitment — which is a **_compulsary_** requirement — for contributors & maintainers to follow a set of guidelines which is listed [here](https://hacktoberfest.com/participation/).

This repository does not fall into the guidelines that were initially set by DigitalOcean. More specifically, this statement: "**_PR/MRs should be useful to maintainers. Repos that encourage simplistic PR/MRs (like adding a name or profile to a list or arbitrarily curating content) will be excluded from Hacktoberfest._**"

^ If you would like me to be even more specific - it's these 3 words: **_arbitrarily curating content_**

This repository was created for the sole purpose of allowing people to contribute pull requests (PRs) with no real purpose other than to be in the "minimum 4 pull requests (PRs)" window to be eligible for a T-Shirt from DigitalOcean.

This 1 week (7 days) of waiting time was implemented so that DigitalOcean can review more than 500K pull requests to ensure that each pull request (PR) meets the guidelines set. Lots of human moderators were chosen by DigitalOcean to take care of this.

If a particular repository was never reported to DigitalOcean where the sole purpose was to defeat the system - it's always — 50% to 75% — possible for your pull requests (PRs) to be flagged and have it **_not_** counted towards your contributions for Hacktoberfest 2022.

As always, humans are humans - and not robots. This means that it's sometimes possible for these pull requests (PRs) to slip through DigitalOcean's reviews and have your pull requests (PRs) counted towards your contributions for Hacktoberfest 2022.

Please be aware that if you made multiple repositories which were all excluded - it's better to be **_safe_** than sorry to say that it's always possible for DigitalOcean to blacklist your GitHub / GitLab account where any pull request (PR) made by other contributors to your repositories will not have their contributions counted towards Hacktoberfest 2022.

DigitalOcean is always looking to implement more safeguarding measures to prevent unfair advantage towards other people and every October in each year, those rules tend to expand.

**Comment by AbhishekTiwari23** [comment]
> There is a commitment — which is a **_compulsary_** requirement — for contributors & maintainers to follow a set of guidelines which is listed [here](https://hacktoberfest.com/participation/).
> 
> This repository does not fall into the guidelines that were initially set by DigitalOcean. More specifically, this statement: "**_PR/MRs should be useful to maintainers. Repos that encourage simplistic PR/MRs (like adding a name or profile to a list or arbitrarily curating content) will be excluded from Hacktoberfest._**"
> 
> ^ If you would like me to be even more specific - it's these 3 words: **_arbitrarily curating content_**
> 
> This repository was created for the sole purpose of allowing people to contribute pull requests (PRs) with no real purpose other than to be in the "minimum 4 pull requests (PRs)" window to be eligible for a T-Shirt from DigitalOcean.
> 
> This 1 week (7 days) of waiting time was implemented so that DigitalOcean can review more than 500K pull requests to ensure that each pull request (PR) meets the guidelines set. Lots of human moderators were chosen by DigitalOcean to take care of this.
> 
> If a particular repository was never reported to DigitalOcean where the sole purpose was to defeat the system - it's always — 50% to 75% — possible for your pull requests (PRs) to be flagged and have it **_not_** counted towards your contributions for Hacktoberfest 2022.
> 
> As always, humans are humans - and not robots. This means that it's sometimes possible for these pull requests (PRs) to slip through DigitalOcean's reviews and have your pull requests (PRs) counted towards your contributions for Hacktoberfest 2022.
> 
> Please be aware that if you made multiple repositories which were all excluded - it's better to be **_safe_** than sorry to say that it's always possible for DigitalOcean to blacklist your GitHub / GitLab account where any pull request (PR) made by other contributors to your repositories will not have their contributions counted towards Hacktoberfest 2022.
> 
> DigitalOcean is always looking to implement more safeguarding measures to prevent unfair advantage towards other people and every October in each year, those rules tend to expand.

Yes, please have a look at this repo because I had made a pr by changing 22 files and adding 100+ lines of code. And now I am feeling that I time was wasted please accept this one as a genuine repo

**Comment by singhrishabh93** [comment]
> > There is a commitment — which is a **_compulsary_** requirement — for contributors & maintainers to follow a set of guidelines which is listed [here](https://hacktoberfest.com/participation/).
> > 
> > This repository does not fall into the guidelines that were initially set by DigitalOcean. More specifically, this statement: "**_PR/MRs should be useful to maintainers. Repos that encourage simplistic PR/MRs (like adding a name or profile to a list or arbitrarily curating content) will be excluded from Hacktoberfest._**"
> > 
> > ^ If you would like me to be even more specific - it's these 3 words: **_arbitrarily curating content_**
> > 
> > This repository was created for the sole purpose of allowing people to contribute pull requests (PRs) with no real purpose other than to be in the "minimum 4 pull requests (PRs)" window to be eligible for a T-Shirt from DigitalOcean.
> > 
> > This 1 week (7 days) of waiting time was implemented so that DigitalOcean can review more than 500K pull requests to ensure that each pull request (PR) meets the guidelines set. Lots of human moderators were chosen by DigitalOcean to take care of this.
> > 
> > If a particular repository was never reported to DigitalOcean where the sole purpose was to defeat the system - it's always — 50% to 75% — possible for your pull requests (PRs) to be flagged and have it **_not_** counted towards your contributions for Hacktoberfest 2022.
> > 
> > As always, humans are humans - and not robots. This means that it's sometimes possible for these pull requests (PRs) to slip through DigitalOcean's reviews and have your pull requests (PRs) counted towards your contributions for Hacktoberfest 2022.
> > 
> > Please be aware that if you made multiple repositories which were all excluded - it's better to be **_safe_** than sorry to say that it's always possible for DigitalOcean to blacklist your GitHub / GitLab account where any pull request (PR) made by other contributors to your repositories will not have their contributions counted towards Hacktoberfest 2022.
> > 
> > DigitalOcean is always looking to implement more safeguarding measures to prevent unfair advantage towards other people and every October in each year, those rules tend to expand.
> 
> Yes, please have a look at this repo because I had made a pr by changing 22 files and adding 100+ lines of code. And now I am feeling that I time was wasted please accept this one as a genuine repo

@AbhishekTiwari23 K understand your feelings. Every contributor of this repo is feeling same. Okay I will contact Hacktoberfest regarding this and try to figure it out asap. Don't worry you have contributed to a good project. A tshirt 👕 can't define your hard work. 
Together we can build something GREAT
Best of luck and you'll always invited to contribute on my projects.
You can connect with me on twitter.
